frontend/components/Pages/AddBackupWalletViaSafePage/index.tsx

```diff
@@ -15,11 +15,12 @@ const { Text } = Typography;
  * update as needed; check https://app.safe.global/new-safe/create for prefixes
  */
 const safeChainPrefix = {
-  [EvmChainId.Ethereum]: 'eth',
+  [AllEvmChainId.Ethereum]: 'eth',
```

Comment:
why do you think we need two separate sets of evmChainIds? wouldn't that be confusing?